docs/notes/2.23.x.md

```diff
@@ -151,6 +151,80 @@ exclude = ["**/tests/*"]
 
 The backend linter will also load a Trufflehog [configuration file](https://github.com/trufflesecurity/trufflehog?tab=readme-ov-file#regex-detector-example) (passed via `trufflehog -c trufflehog-config.yaml`), as long as the configuration file is placed in the root of your codebase with filename: `trufflehog-config.yaml`
 
+#### Python
```

Comment:
This diff seems to have got much larger (moving several paragraphs), which seems a bit surprising. Was that intentional?

Reply:
Yes, there were merge conflicts and while resolving them I realized that the order didn't make much sense. It's just a straight move though.

Reply:
I figured ordering roughly by popularity made more sense than the completely arbitrary previous order, or at least Python then JVM then (everything else).

Reply:
FWIW, the ordering was intentionally "NEW" first, and then alphabetical.

Reply:
Oh, huh. New first I agree with and kept, alphabetical seems less useful, but I can go for that. It is at least objective.